test/unittests/mongoBackend/mongoUpdateContextGeo_test.cpp

```diff
@@ -207,6 +207,7 @@ TEST(mongoUpdateContextGeoRequest, newEntityLocAttribute)
     EXPECT_FALSE(a2.hasField("creDate"));
     EXPECT_FALSE(a2.hasField("modDate"));
     EXPECT_STREQ("A1", ent.getObjectField("location").getStringField("attrName"));
+    ASSERT_TRUE(ent.hasField("location"));
```

Comment:
OK, that's what I'm talking about :-)
Assert to **avoid** segfaults.
Misunderstanding,

NTC